config/model_configs/prognostic_edmfx_aquaplanet.yml

```diff
@@ -2,8 +2,6 @@
 z_max: 30000.0
 z_elem: 10
 dz_bottom: 500.0
-rayleigh_sponge: false
```

Comment:
are these taken out because they will be specified in the Coupler? or because the sponges aren't needed for the aquaplanet